core/common/src/main/java/alluxio/underfs/Fingerprint.java

```diff
@@ -146,6 +146,18 @@ public void updateTag(Tag tag, String value) {
     }
```

Comment:
Should this call `putTag` or we can call this `putTag` and get rid of `putTag` altogether?

Reply:
Yeah, I made this the real `putTag`. Thanks.